content/en/getting_started/synthetics/private_location.md

```diff
@@ -41,48 +43,50 @@ The private location worker is available on Docker Hub:
 
 1. Set up a [Vagrant Ubuntu 16.04 virtual machine][2].
```

Comment:
Users can install Docker on virtually any machine they want, how about saying something like "Install Docker on the machine you would like. You can also install it on a virtual machine like Vagrant Ubuntu 16.04 to get started quickly."? 
We've had several users believe private locations were only supported on Ubuntu VMs because of that sentence so it'd be great if we could clarify that 😅